sycl/include/sycl/ext/oneapi/experimental/annotated_ptr/annotated_ptr.hpp

```diff
@@ -78,6 +78,10 @@ template <typename T, typename... Props>
 class annotated_ref<T, detail::properties_t<Props...>> {
   using property_list_t = detail::properties_t<Props...>;
 
+  static_assert(
+      std::is_trivially_copyable_v<T>,
+      "annotated_ref can only be encapsulated with trivially-copyable type!");
```

Comment:
Is "encapsulated" the correct word here? If anything wouldn't it be "annotated_ref can only encapsulate a trivially-copyable type"? But I think "annotated_ref can only refer to a trivially-copyable type" or "can be only instantiated for" would be better. Same below for annotated_ptr message.